src/components/10-atoms/fieldset/index.js

```diff
@@ -19,6 +19,7 @@ class AXAFieldset extends NoShadowDOM {
       horizontal: { type: Boolean, reflect: true },
       error: { type: String, reflect: true },
       slot: { type: String },
+      noStretchOnMobile: { type: Boolean, reflect: true },
```

Comment:
Just "onMobile"? Or even on Tablet? .... Another name would be better i think. something like "disableResponsiveStretch" or... ?? dont know. I would remove that "mobile" word

Reply:
Good idea👍🏼